nyc/static/css/city_custom.css

```diff
@@ -2,6 +2,7 @@
   --navy-blue-color: #2f56a6;
   --govt-gray-color: #767676;
   --salmon-color: #EB6864;
+  --failure-color: #545454;
 }
```

Comment:
Is there any chance we might use this color elsewhere? It seems possible. So, let's give it a generic name (e.g., "smoky-gray-color", rather than something specific to the "failure" label.

Reply:
Agreed!